### ISSUE TITLE
tensorlayer should add SELU module 

### ISSUE THREAD

**Issue by 20chase** [tensorlayer/tensorlayer#277]
[SELU](https://arxiv.org/pdf/1706.02515.pdf) has a amazing performance nearly, and Keras already support the SELU. So why not someone write a SELU dropout module for tensorlayer:)

**Comment by zsdonghao** [comment]
TL supports SELU by default, just simply pass `tf.nn.selu` into your layers, like `tf.nn.relu`.

**Comment by 20chase** [comment]
Yes, that's true. I mean the SELU drop module rather than the SELU activation function. Actually, I don't know the different between [SELU dropout](https://github.com/bioinf-jku/SNNs/blob/master/selu.py) module and dense(selu) + dropout.
